indexing-service/src/main/java/io/druid/indexing/overlord/TaskRunnerWorkItem.java

```diff
@@ -82,10 +83,7 @@ public DateTime getQueueInsertionTime()
     return queueInsertionTime;
   }
 
-  public TaskRunnerWorkItem withQueueInsertionTime(DateTime time)
-  {
-    return new TaskRunnerWorkItem(taskId, result, createdTime, time);
-  }
+  public abstract TaskLocation getLocation();
```

Comment:
@drcrallen I removed the Comparable interface and everything still compiled… I guess it's not used for anything.